src/pages/home/Home.js

```
@@ -1,10 +1,142 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Accordion from '@material-ui/core/Accordion';
+import AccordionSummary from '@material-ui/core/AccordionSummary';
+import AccordionDetails from '@material-ui/core/AccordionDetails';
+import Typography from '@material-ui/core/Typography';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import './Home.css'
-import '../Page.css'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: '98%',
+    margin: '1%',
+  },
+  heading: {
+    display: "flex",
+    fontSize: theme.typography.pxToRem(15),
+    fontWeight: theme.typography.fontWeightRegular,
+  },
+}));
 
 function Home() {
+  const classes = useStyles();
+  const [xana, setXana] = useState(null);
+  const [serv, setServ] = useState(null);
+  const [app, setApp] = useState(null);
+
+  useEffect(() => {
+    fetch("https://api.server.vincipit.com/")
+      .then(
+        (serv) => {
+          setServ(serv);
+        }
+      )
+    fetch("https://dashboard.vincipit.com/")
```

setup catch strategy instead of ignoring error thrown